jrnl/Entry.py

```diff
@@ -105,6 +108,18 @@ def pprint(self, short=False):
         )
 
         if not short and self.journal.config["linewrap"]:
+            columns = self.journal.config["linewrap"]
+
+            if columns == "auto":
+                try:
+                    columns = os.get_terminal_size()[0]
+                except:  # noqa: E722
```

Comment:
Could you use `except OSError` here? The docs for `get_terminal_size` say that's the exception that fires when it can't get the terminal size for standard out, I'd like to avoid generic exceptions.